.gitignore

```diff
@@ -2,3 +2,4 @@ _site
 .sass-cache
 _gh-pages
 .DS_Store
+.Ulysses-Group.plist
```

Comment:
What's this? Doesn't seem project specific, probably belongs in global git ignore?